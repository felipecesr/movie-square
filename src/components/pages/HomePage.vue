<template>
  <home-template/>
</template>

<script>
import HomeTemplate from '@templates/HomeTemplate.vue';

export default {
  components: { HomeTemplate }
}
</script>

<style>
</style>

