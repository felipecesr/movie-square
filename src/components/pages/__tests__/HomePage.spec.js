import { shallowMount } from '@vue/test-utils';
import HomePage from '../HomePage.vue';

describe('HomePage.vue', () => {
  it('renders correctly', () => {
    expect(shallowMount(HomePage).element).toMatchSnapshot();
  });
});
