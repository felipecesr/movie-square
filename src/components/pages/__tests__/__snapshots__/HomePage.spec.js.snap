// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage.vue renders correctly 1`] = `<home-template-stub />`;
